### DISCUSSION TITLE
Support for WSL (both versions)

### DISCUSSION THREAD

**Discussion by imutyshev** [great-expectations/great_expectations#5167] [Community Use Cases]
It seems that GE is not working well on windows subsystem for linux (checked both versions). Process goes fine untill I try to execute:
context.build_data_docs()
context.open_data_docs(validation_result_identifier)

those commands throw an error:
find: '/mnt/c/Users/\210\341\252\240\255\244\245\340/AppData/Local/Microsoft/WindowsApps/CanonicalGroupLimited.Ubuntu20.04onWindows_79rhkp1fndgsc': No such file or directory

**Comment by andrescevp** [comment]
up this

**Comment by talagluck** [comment]
Hi @imutyshev - thank you for raising this issue! It appears that this has to do with the way Windows process paths with backslashes as compared to the way Unix/Mac systems processes paths with forward slashes. It is difficult for us to prioritize Windows issues internally, but we always welcome contributions. Is this something in which you might be interested? I'm happy to offer PR review as well as any additional guidance you may need.

**Comment by AFineDayFor** [comment]
This would be an excellent topic for a community discussion. it's interesting, and something the community would be more empowered to address!
